components/Modal/modal.css

```diff
@@ -0,0 +1,158 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Julius+Sans+One&display=swap');
+
+:root{
+    --dark-red:#dc3545;
+    --dark-blue:#1e74b9;
+    --white:#fefefe;
+    --dark-orange:#db7e03;
+    --dark-green:#29A744;
+    --container-border-white:#DDDDDC;
+    --container-background-white:#F5F6F8;
+    --violet:rgb(97, 20, 97);
+    --light-pink:#d8bfd8;
+    --card-body-border-white:#E9E8E9;
+    --card-icon-black:rgba(0, 0, 0, 0.54);
+    --card-purple:#9501EE;
+    --background-gray:#c2c5c2;
+    --shadow-black:rgba(0,0,0,0.2);
+    --font-family:'Roboto', sans-serif;
+    --hot-pink:#991869;
+    --brand-font:'Julius Sans One', sans-serif;
+    --black:#000000;
+    --alert-close-brown:#817462;
+    --darker-green:#1A7152;
+    --alert-close-hover-black:rgb(37, 37, 37);
+    --light-orange:#ffe7c5;
+    --light-red:#fcc5c5;
+    --light-blue:#93d2e2;
+    --light-green:#85F1C8;
+    --alert-background-white:#FFFEFF;
+}
+
+*{
+    margin:0;
+    padding:0;
+    box-sizing: border-box;
+    letter-spacing: 0.5px;
+}
+
+.modal{
+    width: 300px;
+  max-width: 100%;
+  height: 250px;
+  max-height: 100%;
+    margin:1rem;
+    border-radius:0.3rem;
+    display: flex;
+    align-items: center;
+    padding: 1rem;
+    background-color: var(--card-body-border-white);
+    flex-wrap: wrap;
+    position: relative;
+    font-family: var(--font-family);
+    flex-direction: column;
+    position: fixed;
+  top: 50%;
```

Comment:
Structure your code properly.
Rest everything looks good.
Good to merge

Reply:
Thank you for pointing it out, i will do that from next time